payments-ui-core/src/main/java/com/stripe/android/paymentsheet/forms/PaymentMethodRequirements.kt

```diff
@@ -37,7 +39,7 @@ data class PaymentMethodRequirements(
      *  - Only [PIRequirement]s are allowed in this set.
      * - If this is null, PaymentIntents (even if SFU is set) are not supported by this LPM.
      */
-    val piRequirements: Set<PIRequirement>?,
+    var piRequirements: Set<PIRequirement>?,
```

Comment:
Why was this changed to var?

Reply:
Reverted the change for now.  This will need to be modified because we need to add delayed based on what the json spec describes.